demo/hello.ts

```diff
@@ -360,12 +360,20 @@ export class HelloApp {
         },
       },
       {
-        key: 'nested.arrays.0',
+        key: 'nested.arrays.0.item',
+        type: 'input',
+        templateOptions: {
+          label: 'Array with dots',
+        },
+        defaultValue: 'Default Value A',
+        parsers: [(value) => (value || '').toUpperCase()],
+      }, {
+        key: 'nested.arrays[1].item',
```

Comment:
I just wanted to keep it consistant with angular-formly, also most developers are used to use [] for arrays